BlockSettleUILib/UiUtils.cpp

```diff
@@ -620,6 +620,13 @@ QValidator::State UiUtils::ValidateDoubleString(QString &input, int &pos, const
 
    QString tempCopy = UiUtils::NormalizeString(input);
 
+   QStringList list = tempCopy.split(defaultDecimalsSeparatorChar);
+   if (list.size() == 2 && list[1].length() > decimals && pos == input.length()) {
```

Comment:
Not sure about "pos == input.length()". What if user type 0.11, move cursor to position 3 ( 0.|11 ) and paste numbers. pos will not be equal to input length

Reply:
Yes, that won't work but I'm not trying to fix that. I just want fix case when user is pasting value with too many decimal digits to empty field, that's all. And without this `pos` check it's possible to enter digits in the middle when field is full (and this looks strange).